src/utils/update/managing.officer.mapper.ts

```diff
@@ -69,6 +71,26 @@ export const splitNames = (officerName: string): string[] => {
   }
 };
 
+export const splitNationalities = (officerNationalities: string | undefined): string[] => {
+  if (officerNationalities === undefined) {
+    return ["", ""];
+  } else {
+    const nationalities = officerNationalities.split(",");
```

Comment:
just 
if (!nationalities) return [""];
return nationalities.length > 1?  nationalities.slice(0, 2) else nationalities;

Reply:
This won't work as it returns a string as opposed to a list of strings

Reply:
Sorry? what returns a string?

Reply:
The task is to split a "," separated string and use the first item for "nationality" and second item for "second nationaily" and ignore any others. So just:

const  firstAndSecondN = "american, french".split(",").slice(0, 2);
const firstN = firstAndSecondN[0]=== undefined? "": firstAndSecondN[0].trim();
const secondN = firstAndSecondN[0]  === undefined ? "": firstAndSecondN[1].trim();